packages/clerk-js/src/ui.retheme/primitives/Button.tsx

```diff
@@ -69,9 +73,18 @@ const { applyVariants, filterProps } = createVariants((theme, props: OwnProps) =
         solid: {
           backgroundColor: vars.accent,
           color: theme.colors.$colorTextOnPrimaryBackground,
-          '&:hover': { backgroundColor: vars.accentDark },
+          ...common.buttonShadow(theme),
+          // '&:hover': { backgroundColor: vars.accentDark }, // TODO: once we have the new color palette
           '&:focus': props.hoverAsFocus ? { backgroundColor: vars.accentDark } : undefined,
-          '&:active': { backgroundColor: vars.accentDarker },
+          // '&:active': { backgroundColor: vars.accentDarker }, // TODO: once we have the new color palette
+          ':after': {
+            position: 'absolute',
+            content: '""',
+            borderRadius: 'inherit',
+            zIndex: -1,
+            inset: 0,
+            background: 'linear-gradient(180deg, rgba(255, 255, 255, 0.10) 0%, rgba(255, 255, 255, 0.00) 100%)',
```

Comment:
Can we also move this literal values?

Reply:
We need to use `colorPrimary` somehow here, in order for this to be themable.

Reply:
Nevermind, didn't understand that correctly. This property actually creates the shadow effect on the button, so no need to change anything